ckanext/ontario_theme/organization.py

```diff
@@ -0,0 +1,85 @@
+from ckan.logic import NotAuthorized, ValidationError
+import ckan.lib.base as base
+import ckan.lib.helpers as h
+import ckan.model as model
+from ckan.common import g, request, _
+from ckan.views.group import _check_access, _action, _get_group_template, set_org
+
+
+def index(group_type='organization', is_organization=True):
```

Comment:
This fn looks the same as core ckan `ckan/views/group.py`. Can you use that instead of making a copy here?

Reply:
No, this function is modified to remove the pagination and to return all fields for the organization so that title_translated can be sorted.